docs/UserGuide.md

```diff
@@ -71,19 +71,19 @@ Displays a message with a list of all available commands and their format.
 
 Format: `help`
 
-### Adding a person: `add`
+### Adding an order: `add`
 
-Adds a person to the address book.
+Adds an order to the address book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG] d/DELIVERY_DATE o/ORDER_DESCRIPTION`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A person can have any number of tags (including 0)
 </div>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 d/13-05-2021 /o strawberry cake 3`
+* `add n/Betsy Crowe p/90123515 e/betsycrowe@example.com a/Newgate Prison p/1234567 t/friend d/13-05-2100 /o chocolate cake 5 durian cake 10 mochi cake 100`
```

Comment:
Not sure if the AB3 allows for 2 p/
I think it does but maybe to make it clearer in the UserGuide, just put one phone number